index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+      <div id="cabecalho">
+      <h1 id="titulo">titulo da página</h1>
```

Comment:
O requisito foi cumprido, mas não acha que ficaria bacana trazer a ideia do projeto nesse cabeçalho?
```suggestion
      <h1 id="titulo">Lições Aprendidas</h1>
```
Ou
```suggestion
      <h1 id="titulo">Lessons Learned</h1>
```
A criatividade pode rolar, mas fica muito interessante se você deixar claro para as pessoas que acessam a página o tipo de conteúdo que elas vão encontrar ali.

Reply:
Muito obrigado pelo feedback @nascjoao. Vou aplicar a sugestão e melhorar mais essa página (y)